Skin/AddOns/Blizzard_Contribution.lua

```diff
@@ -14,6 +14,33 @@ end ]]
 end ]]
 
 function private.AddOns.Blizzard_Contribution()
+	local frame = _G.ContributionCollectionFrame
+	F.SetBD(frame)
+	F.ReskinClose(frame.CloseButton)
```

Comment:
Were these changes tested? I ask because `F` is not defined in this file due to it's API being deprecated. Please use the newer skinning API and paradigm.

There are some (very) basic docs in the wiki.

Reply:
Really looking forward to wiki 👍